split_vocab.py

```
@@ -0,0 +1,41 @@
+import argparse
```

Ruff (0.8.2)

`argparse` imported but unused

Remove unused import: `argparse`

(F401)